README.md

```diff
@@ -369,6 +369,21 @@ will give a more in-depth description of the command and available options.
  - `autograder.cli.courses.assignments.submissions.fetch.course.attempts` -- Get all the most recent submissions (code and grading output) for an assignment.
  - `autograder.cli.courses.users.list` -- List all the users in a course.
 
+#### Proxy Submissions
```

Comment:
I think you should write this entire section just with proxy submissions (proxy.submit) in mind.
I can see in the last paragraph (and list) you plan for expanding it to future commands,
but since I think this family of commands will see some important use, I think each command should be a first class citizen that has it's own entry in the table of contents.

So, I think this section should also include sample usage.